chart/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: ssl-exporter
+description: A Helm chart for ribbybibby/ssl_exporter
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
I think we should start from like 1.0.0, we can discuss it.

Reply:
According to [SemVer](https://semver.org/#spec-item-4)
> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable

We can use 1.0.0, I don't think we will have any significant changes. I used 0.1.0 just because it's the default one used by `helm create`

Reply:
Ok, changed to `1.0.0`